openpype/hosts/unreal/plugins/publish/validate_sequence_frames.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+
+import unreal
+
+import pyblish.api
+
+
+class ValidateSequenceFrames(pyblish.api.InstancePlugin):
+    """Ensure the sequence of frames is complete
+
+    The files found in the folder are checked against the frameStart and
+    frameEnd of the instance. If the first or last file is not
+    corresponding with the first or last frame it is flagged as invalid.
+    """
+
+    order = pyblish.api.ValidatorOrder
+    label = "Validate Sequence Frames"
+    families = ["render"]
+    hosts = ["unreal"]
+    optional = True
+
+    def process(self, instance):
+        self.log.debug(instance.data)
+
+        representations = instance.data.get("representations")
+        for repr in representations:
+            frames = []
+            for x in repr.get("files"):
+                # Get frame number. The last one contains the file extension,
+                # while the one before that is the frame number.
+                # `lstrip` removes any leading zeros. `or "0"` is to tackle
+                # the case where the frame number is "00".
+                frame = int(str(x).split('.')[-2])
+                frames.append(frame)
+            frames.sort()
```

Comment:
For consistency with other areas of OpenPype code base it might make more sense to rely on `clique` for sequences. No?

In this case (where you require the frame number between two dots) you can use exactly [this logic](https://github.com/pypeclub/OpenPype/pull/2875/files#diff-dfd3bf8586623379142bdb58490bf540cca132c24f8fdfd8104095fc66698797R24-R26):

Something like this:
```python
patterns = [clique.PATTERNS["frames"]]
collections, remainder = clique.assemble(files, minimum_items=1,
                                         patterns=patterns)
assert not remainder, "Must not have remainder"
assert len(collections) == 1, "Must detect single collection"
collection = collections[0]
frames = list(collection.indexes)
```

Reply:
Done!